app/views/sessions/new.html.erb

```diff
@@ -2,9 +2,9 @@
 
 <%= form_tag "/login", method: :post do %>
   <%= label_tag :email %><br>
-  <%= text_field_tag :email, placeholder: "Email" %><br>
+  <%= text_field_tag :email, "Email" %><br>
   <%= label_tag :password %><br>
-  <%= text_field_tag :password, placeholder: "Password" %><br>
+  <%= text_field_tag :password, "Password" %><br>
```

Comment:
Removed "placeholder" as it was literally rendering in the form (checked the localhost).

Reply:
Excellent catch! I'm glad you checked this via rails server. We should probably add that into our flow - test are passing, go check rails server localhost!

_Should we also do this for the view users partial _form.html.erb?_

Reply:
I checked the other partial view and the placeholders are rendering accurately. I think the second argument in a form tag is the default value and since we have the actual values in the partial form (ex: @user.name), they are good to go.

Reply:
Ohhhhhh. Got it - thanks for checking into that!